cli/cli.js

```diff
@@ -122,6 +122,11 @@ withCommonRunOptions(program.command("watch"))
 
 withCommonRunOptions(program.command("run <cart>"))
     .description("Open a cartridge in the web runtime")
+    .addOption(
+        new Option("--settle-time <time>", "Changes to the cart must have stopped for at least this long before a reload happens. Increase this if you're getting erroneous double-reloads and cart corruption, decrease it if you want snappier reloads.  In milliseconds.")
+        .env("W4_SETTLE_TIME")
+        .default(500)
```

Comment:
500ms seems a bit long here. Have you been seeing errors with lower values?

Reply:
I agree that keeping this short is important, but I was seeing stray reloads and occasional corruption when it was set at 250ms, I think caused by the separate `wasm-opt` step. I thought it was better to have a safe default and let people with fast machines tune it down. I've just tested again, and on my setup 300ms seems pretty stable. I'll change it to that.